docs/aws.md

```diff
@@ -31,16 +31,16 @@ This section describes how to use Iceberg with AWS.
 
 ## Enabling AWS Integration
 
-The `iceberg-aws` module is bundled with Spark and Flink engine runtimes for all versions from `0.11.0` onwards.
+The `iceberg-aws` module is bundled with Spark and Flink engine runtimes for all versions from `0.11.0` onwards.ƒ
```

Comment:
Could you remove the `f` at the end of the line?

Reply:
oops. Thanks for pointing this out!